include/CLine3D.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "TVector3.h"
+
+class CLine3D
+{
+ private:
+  TVector3 _point;
+  TVector3 _direction;
+  TVector3 _u;
+  TVector3 _v;
```

Comment:
We need to standardize the variable names. For now, I would follow the style initially proposed in a minimal way, without introducing a new one for class data members.

https://google.github.io/styleguide/cppguide.html#Variable_Names

which suggests trailing underscores